src/layout/Footer/Footer.js

```diff
@@ -1,11 +1,28 @@
 import React from 'react';
 import logo from '@axa-fr/react-toolkit-core/dist/assets/logo-axa.svg';
-import { Footer } from '@axa-fr/react-toolkit-all';
-
+import { getLabel } from 'services/formatting';
+import { version } from '../../../package.json';
 import './Footer.scss';
 
-const FooterApp = () => {
-  return <Footer icon={logo} />;
-};
+const FooterApp = () => (
+  <footer className="footer">
+    <div className="container-fluid container">
+      <div className="af-logo">
+        <img className="af-logo__brand" src={logo} alt="Logo Axa" />
+      </div>
+      <div className="af-footer-content">
+        <div className="footer__content">
+          <div className="footer__base">
+            {getLabel('footer.title')} 
```

Comment:
may be i18n is not a good idea for a starter, few projects need i18n and they will have unecessary code ^^

Reply:
there is not i18n